storage/integration_test.go

```diff
@@ -5728,7 +5751,7 @@ func TestIntegration_PostPolicyV4_WithCreds(t *testing.T) {
 				}
 			})
 		}
-	}, option.WithCredentials(creds))
+	}, option.WithAuthCredentials(creds))
```

Comment:
Did you try running the integration tests without changes? I would like to make sure they pass even using the old auth before switching them over.

Signed URL stuff in particular, do you think it would be useful to test with both the old and new auth paths?